### DISCUSSION TITLE
change burner after charge

### DISCUSSION THREAD

**Discussion by ErezMinuto** [artisan-roaster-scope/artisan#1462] [Q&A]
Want to change burner settings in alarms  x seconds after charging. 
Ideas?

**Comment by MAKOMO** [comment]
At 2:00min after CHARGE set the slider "Burner" to 80% independent on any curve data like bean temperature.

<img width="1028" alt="Screenshot 2024-02-21 at 17 06 53" src="https://github.com/artisan-roaster-scope/artisan/assets/13909011/8de252f4-e849-4fe0-bf52-3515d9cb4430">
